src/engraving/dom/keysig.cpp

```diff
@@ -156,6 +157,15 @@ bool KeySig::isChange() const
     return staff()->currentKeyTick(keyTick) == keyTick;
 }
 
+bool KeySig::isFirstSystemKeySig()
```

Comment:
Ideally, we should make this function static and remove it from this class (perhaps declare it inside utils.h or directly in transpose.cpp). We are trying to make the implementation of engraving items as simple as possible (so they won't implicitly use systems/measures/etc)